src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnClientConfiguration.md

```diff
@@ -0,0 +1,139 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# New-AzureRmVpnClientConfiguration
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```

Comment:
You need to actually fill this out.  This is the customer's primary means of figuring out how the cmdlets work, these descriptions need to be complete and correct, and you need to provide a valid example of usage, with realistic output